ui/pages/confirmations/confirm/confirm.tsx

```diff
@@ -29,6 +30,7 @@ const Confirm = () => {
         ///: END:ONLY_INCLUDE_IF
       }
       <ScrollToBottom>
+        <LedgerInfo />
```

Comment:
we will need to add a padding above this. That, or we could add padding top/bottom to the content container and update paddings for children elements 
<img width="383" alt="Screenshot 2024-04-09 at 10 50 43 PM" src="https://github.com/MetaMask/metamask-extension/assets/20778143/4e1c435f-2682-4a01-b2cd-3f92a8d9064e">